### PR TITLE
Kitchen Storage Tweaks

### PR DESCRIPTION
**Pull request by SleepyGemmy**
tweaks the kitchen storages (fridges, freezers, and cabinets) to have their own paths instead of sharing the freezer's.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

File: code/game/objects/structures/crates_lockers/closets/secure/kitchen.dm [hunk 1]
```diff
@@ -1,115 +1,149 @@
...
+// Empty Refrigerator
+/obj/structure/closet/secure_closet/refrigerator
+	name = "refrigerator"
+	desc = "A white refrigerator."
+	icon_state = "refrigerator"
+	door_anim_squish = 0.22
+	door_anim_angle = 123
+
+// Standard Refrigerator
+/obj/structure/closet/secure_closet/refrigerator/standard/fill()
```
Comment (alsoandanswer) [inline code/game/objects/structures/crates_lockers/closets/secure/kitchen.dm:14]:
```suggestion
/obj/structure/closet/secure_closet/refrigerator/kitchen/fill()
```

Comment (alsoandanswer) [inline code/game/objects/structures/crates_lockers/closets/secure/kitchen.dm:14]:
```suggestion
/obj/structure/closet/secure_closet/refrigerator/kitchen/fill()
```

File: html/changelogs/SleepyGemmy-refrigerators_and_freezers.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (alsoandanswer) [inline html/changelogs/SleepyGemmy-refrigerators_and_freezers.yml:30]:
```suggestion
author: SleepyGemmy
```

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**